pkg/sql/sqlbase/constants.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 // DefaultSearchPath is the search path used by virgin sessions.
-var DefaultSearchPath = sessiondata.MakeSearchPath([]string{"public"})
+var DefaultSearchPath = sessiondata.MakeSearchPath([]string{"public", "pg_extension"})
```

Comment:
Why do we include pg_extension here? I thought the whole point was to have pg_extension implicitly resolve before public?

Reply:
accident!